lib/netconf/netconf.go

```diff
@@ -177,13 +180,13 @@ func Load(path string) (Network, error) {
 }
 
 // Save saves the network configuration to the given path.
-func Save(network Network, path string) error {
+func Save(ctx context.Context, network Network, path string) error {
 	for _, chain := range network.Chains {
 		if chain.IsOmniConsensus {
 			continue
 		}
 		if chain.PortalAddress == (common.Address{}) {
-			return errors.New("empty portal address", "chain", chain.Name)
```

Comment:
can't error since monitor-only doesn't have portal addresses, so just WARN for what it is worth